astroid/bases.py

```diff
@@ -233,7 +233,7 @@ def __str__(self):
 
     def callable(self):
         try:
-            self._proxied.getattr('__call__', class_context=False)
+            self._proxied.getattr('__call__')
```

Comment:
Yeah, but I'm asking from a protocol perspective. ``class_context`` is False, since the lookup is done on the instance, thus it should not go through the metaclass lookup and so on. What I'm asking is why typing.NamedTuple is special. Is it an instance or a class? If it is an Instance, how come it has a ``__call__`` defined in the metaclass (I presume), that it can access in this place? Where does its ``__call__`` come from?